docker/docker-compose.ci.mysql.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2020-2022 Equinix, Inc
+# Copyright 2014-2022 The Billing Project, LLC
+#
+# The Billing Project licenses this file to you under the Apache License, version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at:
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+version: '3.8'
+
+services:
+  killbill:
+    network_mode: host
+    image: killbill/killbill:0.23.0-9116fa4-SNAPSHOT
```

Comment:
Could upgrade to 0.24. now :-)

Reply:
Yup, I will do this after merging to master 👍